spec/install/deploy_spec.rb

```diff
@@ -263,4 +263,28 @@
       should_be_installed "rack 1.0.0"
     end
   end
+
+  context "with path in Gemfile and packed" do
```

Comment:
I'd suggest splitting up and reformatting these new specs instead of having them all in one `it` block?